tests/integration_tests/test_chainermn.py

```diff
@@ -218,34 +217,6 @@ def objective(_trial: ChainerMNTrial, _comm: bool) -> float:
             failed_trials = [t for t in mn_study.trials if t.state == TrialState.FAIL]
             assert len(failed_trials) == n_trials + 1
 
-    @staticmethod
-    @pytest.mark.parametrize("storage_mode", STORAGE_MODES)
-    def test_relative_sampling(storage_mode: str, comm: CommunicatorBase) -> None:
-
-        relative_search_space = {
-            "x": distributions.FloatDistribution(low=-10, high=10),
-            "y": distributions.FloatDistribution(low=20, high=30, log=True),
-            "z": distributions.CategoricalDistribution(choices=(-1.0, 1.0)),
-        }
-        relative_params = {"x": 1.0, "y": 25.0, "z": -1.0}
-        sampler = DeterministicRelativeSampler(relative_search_space, relative_params)
-
-        with MultiNodeStorageSupplier(storage_mode, comm) as storage:
-            study = TestChainerMNStudy._create_shared_study(storage, comm, sampler=sampler)
-            mn_study = ChainerMNStudy(study, comm)
-
-            # Invoke optimize.
-            n_trials = 20
-            func = Func()
-            mn_study.optimize(func, n_trials=n_trials)
-
-            # Assert trial counts.
-            assert len(mn_study.trials) == n_trials
-
-            # Assert the parameters in `relative_params` have been suggested among all nodes.
-            for trial in mn_study.trials:
-                assert trial.params == relative_params
-
     @staticmethod
     def _create_shared_study(
```

Comment:
Not a strong opinion but the optional `sampler` argument is no longer passed any called, with `test_relative_sampling` removed. How about removing this unused argument from the definition?

Reply:
Sounds reasonable! I removed `sampler` argument at f6f09ac.